rfc/0054-sdk3-management-apis.md

```diff
@@ -395,6 +406,22 @@ void CreateIndex(string bucketName, string indexName, []string fields,  [options
 
   * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
 
+  * `CollectionName` - the name of the collection to restrict indexes to.
+  * `ScopeName` - the name of the scope to restrict indexes to.
+    * If either `CollectionName` or `ScopeName` are set then both *must* be set.
+
+### N1QL
+
+No collection set:
+```
+"CREATE INDEX `indexName` ON `bucketName`"
```

Comment:
Could we specify if this form indexes all collections or only default one?

Reply:
This creates the index only on the default collection.

Reply:
Right, but it is not clear for the description of the method:

> Creates a new index.

And also the `CollectionName` and `ScopeName` description talk about restricting, but they are not really restricting the index, because by default it is already restricted to default collection.